src/expr/macro/src/gen.rs

```diff
@@ -61,8 +61,12 @@ impl FunctionAttr {
         let name = self.name.clone();
         let mut args = Vec::with_capacity(self.args.len());
         for ty in &self.args {
+            if ty == "..." {
+                break;
+            }
```

Comment:
What if `...` is not the last arg? Should we add some error check here?

Btw, can you update docs?

Reply:
It is only allowed at last. You are right. I should add a check here.